packages/backend/src/services/SchedulerService/SchedulerService.ts

```diff
@@ -219,7 +219,36 @@ export class SchedulerService {
             SchedulerModel.getSlackChannels(scheduler.targets),
         );
 
-        await schedulerClient.generateDailyJobsForScheduler(scheduler);
+        // We only generate jobs if the scheduler is enabled
+        if (scheduler.enabled)
```

Comment:
Does checking for 'enabled' vs using a disabled flag mean that exisitng schedulers won't run unless they get enabled=true added to the database?

I imagine you have a good reason for doing it this way, but I would have assumed the new flag would be 'disabled' because the default state of schedulers seems like it would enabled.

Reply:
I think that does make sense, but what I mean is more like, why use a flag that must be defined in order for them to run instead of a flag that must be defined in order to turn them off -- does that make sense? If we add 'disabled' instead, all existing schedulers won't have to change. Their behavior will only change when someone disables them. With 'enabled' we'll have to add the entry to the database for each existing scheduler, right?  

Maybe I'm missing some magic you know about 😄

Reply:
> 'enabled' we'll have to add the entry to the database for each existing scheduler, right?

We already do that, on the migration, and it defaultsTo (true) . SO we don't need to manually change anything about existing schedulers, all existing schedulers will be enabled by default. And they can only be disabled by clicking the toggle